CppCoreGuidelines.md

```diff
@@ -6710,7 +6710,7 @@ In particular, ensure that an object compares equal to its copy.
     {
         Sorted_vector<string> v2 {v};
         if (v != v2)
-            cout << "insanity rules!\n";
+            cout << "What's happening here?\n";
```

Comment:
I think the point here was to emphasize that such code behaves contrary to reason and logic.

Reply:
@cubbimew: I am open to other suggestions, but usage of "insane" is inappropriate for such emphasis according to guides on writing, see the link in the description of the pull request.

Reply:
@cubbimew would a better way to express that be
```cout << "This code behaves contrary to reason and logic\n";```

Reply:
I agree, I changed to slightly shorter `cout << "Behavior against reason and logic.\n"`.